### PR TITLE
Refactors bruise packs and ointment, with a mild rebalance to them and patches added

### PR DESCRIPTION
**Pull request by ivanmixo**
## About The Pull Request

Makes bruise pack/ointment/gauze code less sucky. You also can't use them on stuff that isn't hurt anymore, nice.

You can't stack the do_afters anymore when using them. This means you can't outheal space anymore by stacking the bars.

The area of the body you're targeting when applying a patch/medspray now actually matters. The `reaction_mob` proc will try to use the limb you're targeting as the limb it'll apply the instant damage healing property to when loaded with chems like styptic powder/synthflesh/silver sulf. 

Killed off the useless `do_after_mob` proc and replaced all instances with `do_mob`

## Why It's Good For The Game

Better code gooder, bruise packs and ointment being slightly less OP also good.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

image

image

image

image

image

image

Applied 30u synthflesh to chest.

image

</details>

## Changelog
:cl:
balance: bruise pack/gauze/ointment application do_afters(the use bars) can't be stacked on yourself anymore yet
fix: you can't apply gauze on people who arent hurt anymore
del: useless do_after_mob proc
refactor: refactored bruise pack/ointment/gauze application proc
balance: the area of the body you're targeting when applying touch based healing reagents through a patch/medspray now actually matters
code: Added a bunch of comments to items related to chems and stacks
/:cl:

### CODE REVIEW

**Comment by Coding-Grape** [comment]
I like this, it might incentivise actually making styptic powder and silver sulf patches instead of hee hoo ointment and bruise packs

File: code/__HELPERS/mobs.dm [hunk 1]
```diff
@@ -186,43 +186,49 @@ GLOBAL_LIST_EMPTY(species_list)
...
+		return FALSE
 	var/user_loc = user.loc
 
 	var/drifting = FALSE
 	if(SSmove_manager.processing_on(user, SSspacedrift))
-		drifting = 1
+		drifting = TRUE
+
+	LAZYADD(user.do_afters, target)
+	LAZYADD(target.targeted_by, user)
```
Comment (PowerfulBacon) [inline code/__HELPERS/mobs.dm:197]: I hate that this is added specifically just for brute/burn kits, especially since this add event more references. I'm sure there are other things in the game which don't let you use multiple at once.

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]

File: code/__HELPERS/mobs.dm (hunk 1 above)
Comment (ivanmixo) [reply code/__HELPERS/mobs.dm:197]: It was an oversight that it didn't have these in the first place.

Comment (ivanmixo) [reply code/__HELPERS/mobs.dm:197]: For reference I added the `do_afters` list to mobs in the blind people can examine stuff PR, but I forgot to add it to `do_mob` as well. 

If there are other places like this they should be converted to use the `do_afters` list instead of whatever they're doing.

Comment (PowerfulBacon) [inline code/__HELPERS/mobs.dm:197]: If the user gets deleted in the middle of this, won't target.targeted_by have a deleted user in it and cause a hard-del?

File: code/datums/components/crafting/recipes.dm [hunk 2]
```diff
@@ -927,15 +927,15 @@
...
+				/datum/reagent/medicine/styptic_powder = 40)
 	category = CAT_MISC
 
 /datum/crafting_recipe/burn_pack
 	name = "Burn Ointment"
 	result = /obj/item/stack/medical/ointment/one
 	time = 1
 	reqs = list(/obj/item/stack/medical/gauze = 1,
-				/datum/reagent/medicine/silver_sulfadiazine = 10)
+				/datum/reagent/medicine/silver_sulfadiazine = 40)
```
Comment (PowerfulBacon) [inline code/datums/components/crafting/recipes.dm:945]: Undocumented changes

File: code/game/objects/items/miscellaneous.dm [hunk 3]
```diff
@@ -215,25 +215,25 @@
...
-				to_chat(M, "<span class='userdanger'>[user] stuffs you into the [src]!</span>")
+			if(kidnapee.mob_size <= capacity)
+				src.contents += kidnapee
+				capacity -= kidnapee.mob_size
+				user.visible_message("<span class='warning'>[user] stuffs [kidnapee] into the [src]!</span>")
+				to_chat(kidnapee, "<span class='userdanger'>[user] stuffs you into the [src]!</span>")
 			else
-				to_chat(user, "[M] will not fit in the tophat!")
+				to_chat(user, "[kidnapee] will not fit in the tophat!")
 	else if (isitem(target))
```
Comment (PowerfulBacon) [inline code/game/objects/items/miscellaneous.dm]: Unrelated changes

File: code/game/objects/items/stacks/medical.dm [hunk 4]
```diff
@@ -7,98 +7,86 @@
...
-					to_chat(user, "<span class='warning'>[H] isn't bleeding!</span>")
-					return
-
-
-	if(isliving(M))
-		if(!M.can_inject(user, 1))
+		if(critter.health == critter.maxHealth)
+			to_chat(user, "<span class='notice'>[M] is at full health.</span>")
 			return
+		if(heal_brute < 1)
```
Comment (PowerfulBacon) [inline code/game/objects/items/stacks/medical.dm]: Should be less than the minimum amount of damage precision

File: code/game/objects/items/stacks/medical.dm [hunk 5]
```diff
@@ -7,98 +7,86 @@
...
+				to_chat(user, "<span class='warning'>[H]'s bleeding is already bandaged!</span>")
 				return
-			user.visible_message("<span class='green'>[user] applies [src] on [t_himself].</span>", "<span class='green'>You apply [src] on yourself.</span>")
+			H.suppress_bloodloss(stop_bleeding)
 
+	if(!IS_ORGANIC_LIMB(affecting))
+		to_chat(user, "<span class='warning'>Medicine won't work on a robotic limb!</span>")
+		return
+
+	if(!(affecting.brute_dam || affecting.burn_dam))
```
Comment (PowerfulBacon) [inline code/game/objects/items/stacks/medical.dm:76]: Brute packs shouldn't work on burnt limbs if we are blocking this

File: code/game/objects/items/stacks/medical.dm [hunk 6]
```diff
@@ -108,8 +96,7 @@
 	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
 	heal_brute = 40
-	self_delay = 20
-	grind_results = list(/datum/reagent/medicine/styptic_powder = 10)
+	grind_results = list(/datum/reagent/medicine/styptic_powder = 40)
```
Comment (PowerfulBacon) [inline code/game/objects/items/stacks/medical.dm:103]: Related to the undocumented changes

File: code/datums/components/crafting/recipes.dm (hunk 2 above)
Comment (PowerfulBacon) [reply code/datums/components/crafting/recipes.dm:945]: Also 40 is an insane amount for 1 ointment and makes them even less worth making

File: code/game/objects/items/stacks/stack.dm [hunk 7]
```diff
@@ -76,26 +88,26 @@
 		return ..()
 	if(amount <= (max_amount * (1/3)))
 		icon_state = initial(icon_state)
-	else if (amount <= (max_amount * (2/3)))
+	else if(amount <= (max_amount * (2/3)))
 		icon_state = "[initial(icon_state)]_2"
 	else
 		icon_state = "[initial(icon_state)]_3"
-	..()
+	return ..()
```
Comment (PowerfulBacon) [inline code/game/objects/items/stacks/stack.dm:95]: Does update_icon actually return?

File: code/game/objects/items/stacks/stack.dm [hunk 8]
```diff
@@ -12,21 +12,33 @@
 /obj/item/stack
```
Comment (PowerfulBacon) [inline code/game/objects/items/stacks/stack.dm:12]: Ton of unrelated changes in this file

File: code/game/objects/items/stacks/tiles/light.dm [hunk 9]
```diff
@@ -31,7 +31,7 @@
```
Comment (PowerfulBacon) [inline code/game/objects/items/stacks/tiles/light.dm]: Unrelated changes

File: code/game/objects/items/stacks/tiles/tile_types.dm [hunk 10]
```diff
@@ -488,7 +488,7 @@
 /obj/item/stack/tile/plasteel/cyborg
```
Comment (PowerfulBacon) [inline code/game/objects/items/stacks/tiles/tile_types.dm]: Unrelated changes

File: code/game/objects/structures/crates_lockers/closets.dm [hunk 11]
```diff
@@ -343,22 +343,21 @@
...
+						"<span class='warning'>You [actuallyismob ? "try to " : ""]stuff [O] into [src].</span>", \
+						"<span class='italics'>You hear clanging.</span>")
 	if(actuallyismob)
-		if(do_after_mob(user, targets, 40))
+		if(do_mob(user, O, 40))
 			user.visible_message("<span class='notice'>[user] stuffs [O] into [src].</span>", \
-							 	 "<span class='notice'>You stuff [O] into [src].</span>", \
-							 	 "<span class='italics'>You hear a loud metal bang.</span>")
+								"<span class='notice'>You stuff [O] into [src].</span>", \
+								"<span class='italics'>You hear a loud metal bang.</span>")
```
Comment (PowerfulBacon) [inline code/game/objects/structures/crates_lockers/closets.dm]: Unrelated changes

File: code/game/objects/structures/crates_lockers/closets.dm [hunk 12]
```diff
@@ -369,7 +368,7 @@
 				close()
 	else
 		O.forceMove(T)
-	return 1
+	return TRUE
```
Comment (PowerfulBacon) [inline code/game/objects/structures/crates_lockers/closets.dm:371]: Unrelated changes

File: code/modules/clothing/gloves/miscellaneous.dm [hunk 13]
```diff
@@ -112,16 +112,15 @@
...
-			playsound(M, 'sound/weapons/emitter2.ogg', 25, 1, -1)
-			A.attack_hand(M)
-			return 1
-
+		user.changeNext_move(CLICK_CD_MELEE)
+		if(do_mob(user, A, 8))
+			new /obj/effect/temp_visual/telekinesis(user.loc)
+			playsound(user, 'sound/weapons/emitter2.ogg', 25, 1, -1)
+			A.attack_hand(user)
+			return TRUE
```
Comment (PowerfulBacon) [inline code/modules/clothing/gloves/miscellaneous.dm]: Unrelated changes

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 14]
```diff
@@ -122,7 +122,7 @@
 
 		if(S?.source)
 			S.materials = list()
-			S.is_cyborg = 1
+			S.is_cyborg = TRUE
```
Comment (PowerfulBacon) [inline code/modules/mob/living/silicon/robot/robot_modules.dm]: Unrelated changes

File: code/modules/power/cable.dm [hunk 15]
```diff
@@ -507,7 +507,7 @@ GLOBAL_LIST_INIT(cable_coil_recipes, list (new/datum/stack_recipe("cable restrai
 	usesound = 'sound/items/deconstruct.ogg'
 
 /obj/item/stack/cable_coil/cyborg
-	is_cyborg = 1
+	is_cyborg = TRUE
 	materials = list()
```
Comment (PowerfulBacon) [inline code/modules/power/cable.dm]: Unrelated changes

File: code/modules/reagents/chemistry/holder.dm [hunk 16]
```diff
@@ -584,7 +584,7 @@
 			can_process = TRUE
 	return can_process
 
-/datum/reagents/proc/reaction(atom/A, method = TOUCH, volume_modifier = 1, show_message = 1)
+/datum/reagents/proc/reaction(atom/A, method = TOUCH, volume_modifier = 1, show_message = 1, obj/item/bodypart/affecting)
 	var/react_type
```
Comment (PowerfulBacon) [inline code/modules/reagents/chemistry/holder.dm:588]: Unrelated changes

File: code/modules/reagents/chemistry/reagents.dm [hunk 17]
```diff
@@ -46,16 +46,16 @@ GLOBAL_LIST_INIT(name2reagent, build_name2reagent())
 	. = ..()
 	holder = null
 
-/datum/reagent/proc/reaction_mob(mob/living/M, method=TOUCH, reac_volume, show_message = 1, touch_protection = 0)
+/datum/reagent/proc/reaction_mob(mob/living/M, method = TOUCH, reac_volume, show_message = 1, touch_protection = 0, obj/item/bodypart/affecting)
 	if(!istype(M))
```
Comment (PowerfulBacon) [inline code/modules/reagents/chemistry/reagents.dm:50]: Unrelated changes

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 18]
```diff
@@ -425,17 +427,18 @@
 	metabolization_rate = 2.5 * REAGENTS_METABOLISM
 	overdose_threshold = 125
 
-/datum/reagent/medicine/synthflesh/reaction_mob(mob/living/M, method=TOUCH, reac_volume,show_message = 1)
+/datum/reagent/medicine/synthflesh/reaction_mob(mob/living/M, method=TOUCH, reac_volume, show_message = 1, touch_protection, obj/item/bodypart/affecting)
```
Comment (PowerfulBacon) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm:430]: Undocumented/unrelated changes

File: code/modules/reagents/reagent_containers.dm [hunk 19]
```diff
@@ -2,19 +2,29 @@
 	name = "Container"
```
Comment (PowerfulBacon) [inline code/modules/reagents/reagent_containers.dm:2]: Unrelated changes

File: code/modules/reagents/reagent_containers/medspray.dm [hunk 20]
```diff
@@ -32,11 +32,22 @@
 		amount_per_transfer_from_this = initial(amount_per_transfer_from_this)
 	to_chat(user, "<span class='notice'>You will now apply the medspray's contents in [squirt_mode ? "short bursts":"extended sprays"]. You'll now use [amount_per_transfer_from_this] units per use.</span>")
 
-/obj/item/reagent_containers/medspray/attack(mob/M, mob/user, def_zone)
+/obj/item/reagent_containers/medspray/attack(mob/living/carbon/M, mob/user, def_zone)
```
Comment (PowerfulBacon) [inline code/modules/reagents/reagent_containers/medspray.dm:35]: Unrelated changes

File: code/modules/reagents/reagent_containers/patch.dm [hunk 21]
```diff
@@ -4,28 +4,28 @@
...
+	if(!ishuman(L))
+		return ..()
+	var/obj/item/bodypart/affecting = L.get_bodypart(check_zone(user.zone_selected))
+	if(!affecting)
+		balloon_alert(user, "The limb is missing.")
+		return
+	if(!IS_ORGANIC_LIMB(affecting))
+		balloon_alert(user, "[src] doesn't work on robotic limbs.")
+		return
+	return ..(L, user, affecting)
```
Comment (PowerfulBacon) [inline code/modules/reagents/reagent_containers/patch.dm]: Unrelated changes

File: code/modules/reagents/reagent_containers/pill.dm [hunk 22]
```diff
@@ -27,7 +27,7 @@
 	return
 
 
-/obj/item/reagent_containers/pill/attack(mob/M, mob/user, def_zone)
+/obj/item/reagent_containers/pill/attack(mob/M, mob/user, obj/item/bodypart/affecting)
```
Comment (PowerfulBacon) [inline code/modules/reagents/reagent_containers/pill.dm:30]: Unrelated changes

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]

File: code/game/objects/items/miscellaneous.dm (hunk 3 above)
Comment (ivanmixo) [reply code/game/objects/items/miscellaneous.dm]: Related to the useless proc removal.

File: code/game/objects/items/stacks/stack.dm (hunk 7 above)
Comment (ivanmixo) [reply code/game/objects/items/stacks/stack.dm:95]: Good to be safe.

File: code/game/objects/structures/crates_lockers/closets.dm (hunk 11 above)
Comment (ivanmixo) [reply code/game/objects/structures/crates_lockers/closets.dm]: Related to proc removal.

File: code/modules/clothing/gloves/miscellaneous.dm (hunk 13 above)
Comment (ivanmixo) [reply code/modules/clothing/gloves/miscellaneous.dm]: Related to proc removal.

File: code/modules/reagents/chemistry/holder.dm (hunk 16 above)
Comment (ivanmixo) [reply code/modules/reagents/chemistry/holder.dm:588]: Related to patches caring about which body part you're targeting to do their effects.

File: code/modules/reagents/chemistry/reagents.dm (hunk 17 above)
Comment (ivanmixo) [reply code/modules/reagents/chemistry/reagents.dm:50]: Related to patches caring about which body part you're targeting to do their effects.

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm (hunk 18 above)
Comment (ivanmixo) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm:430]: Related to patches caring about which body part you're targeting to do their effects.

File: code/modules/reagents/reagent_containers/medspray.dm (hunk 20 above)
Comment (ivanmixo) [reply code/modules/reagents/reagent_containers/medspray.dm:35]: Related to patches caring about which body part you're targeting to do their effects.

File: code/modules/reagents/reagent_containers/patch.dm (hunk 21 above)
Comment (ivanmixo) [reply code/modules/reagents/reagent_containers/patch.dm]: It's a... refactor/code adjustment related to patches which I was touching. I need it to be like this to work.

File: code/modules/reagents/reagent_containers/pill.dm (hunk 22 above)
Comment (ivanmixo) [reply code/modules/reagents/reagent_containers/pill.dm:30]: Related to patches caring about which body part you're targeting to do their effects.

File: code/datums/components/crafting/recipes.dm (hunk 2 above)
Comment (ivanmixo) [reply code/datums/components/crafting/recipes.dm:945]: It's to make it equal with brute/burn patches. 1u of styptic is equal to one brute damage. 1u of silver sulfadaizine is equal to one burn damage. They're very much still worth making, no stamina penalty for applying them and you can stack them.

File: code/game/objects/items/stacks/rods.dm [hunk 23]
```diff
@@ -76,7 +76,7 @@ GLOBAL_LIST_INIT(rod_recipes, list ( \
 
 /obj/item/stack/rods/cyborg
 	materials = list()
-	is_cyborg = 1
+	is_cyborg = TRUE
```
Comment (ivanmixo) [inline code/game/objects/items/stacks/rods.dm]:
```suggestion
	is_cyborg = 1
```

File: code/game/objects/items/stacks/sheets/glass.dm [hunk 24]
```diff
@@ -34,7 +34,7 @@ GLOBAL_LIST_INIT(glass_recipes, list ( \
 
 /obj/item/stack/sheet/glass/cyborg
 	materials = list()
-	is_cyborg = 1
+	is_cyborg = TRUE
```
Comment (ivanmixo) [inline code/game/objects/items/stacks/sheets/glass.dm]:
```suggestion
	is_cyborg = 1
```

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 25]
```diff
@@ -142,7 +142,7 @@ GLOBAL_LIST_INIT(metal_recipes, list ( \
 
 /obj/item/stack/sheet/iron/cyborg
 	materials = list()
-	is_cyborg = 1
+	is_cyborg = TRUE
```
Comment (ivanmixo) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]:
```suggestion
	is_cyborg = 1
```

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 26]
```diff
@@ -633,7 +633,7 @@ GLOBAL_LIST_INIT(brass_recipes, list ( \
 
 /obj/item/stack/tile/brass/cyborg
 	materials = list()
-	is_cyborg = 1
+	is_cyborg = TRUE
```
Comment (ivanmixo) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]:
```suggestion
	is_cyborg = 1
```

File: code/game/objects/items/stacks/stack.dm (hunk 8 above)
Comment (ivanmixo) [reply code/game/objects/items/stacks/stack.dm:12]: Is making docs and refactoring the parent of the main subject of the thing I'm refactoring really going to hurt?

File: code/game/objects/items/stacks/tiles/light.dm [hunk 27]
```diff
@@ -31,7 +31,7 @@
 
 /obj/item/stack/tile/light/cyborg
 	materials = list() 
-	is_cyborg = 1
+	is_cyborg = TRUE
```
Comment (ivanmixo) [inline code/game/objects/items/stacks/tiles/light.dm]:
```suggestion
	is_cyborg = 1
```

File: code/game/objects/items/stacks/tiles/tile_types.dm [hunk 28]
```diff
@@ -488,7 +488,7 @@
...
 	desc = "The ground you walk on." //Not the usual floor tile desc as that refers to throwing, Cyborgs can't do that - RR
 	materials = list() // All other Borg versions of items have no Iron or Glass - RR
-	is_cyborg = 1
+	is_cyborg = TRUE
```
Comment (ivanmixo) [inline code/game/objects/items/stacks/tiles/tile_types.dm]:
```suggestion
	is_cyborg = 1
```

File: code/game/objects/structures/crates_lockers/closets.dm (hunk 12 above)
Comment (ivanmixo) [reply code/game/objects/structures/crates_lockers/closets.dm:371]: I'm touching this file anyways, why not.

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 29]
```diff
@@ -122,7 +122,7 @@
...
```
Comment (ivanmixo) [inline code/modules/mob/living/silicon/robot/robot_modules.dm]:
```suggestion
			S.is_cyborg = 1
```

File: code/modules/power/cable.dm [hunk 30]
```diff
@@ -507,7 +507,7 @@ GLOBAL_LIST_INIT(cable_coil_recipes, list (new/datum/stack_recipe("cable restrai
 	usesound = 'sound/items/deconstruct.ogg'
 
 /obj/item/stack/cable_coil/cyborg
-	is_cyborg = 1
+	is_cyborg = TRUE
```
Comment (ivanmixo) [inline code/modules/power/cable.dm]:
```suggestion
	is_cyborg = 1
```

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm (hunk 18 above)
Comment (ivanmixo) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm:430]: If you're worried about the new way of healing damage - it's the same thing but condensed into one line.

File: code/modules/reagents/reagent_containers.dm (hunk 19 above)
Comment (ivanmixo) [reply code/modules/reagents/reagent_containers.dm:2]: Is more documentation on vars really going to kill us?

**Comment by PowerfulBacon** [comment]
> Is making docs and refactoring the parent of the main subject of the thing I'm refactoring really going to hurt?

Update the PR body to say what you are doing then. Refactoring is fine if you say what you are actually changing

**Comment by itsmeow** [comment]
Status of this PR? Requested changes implemented?

File: code/game/objects/items/stacks/medical.dm [hunk 31]
```diff
@@ -39,7 +39,7 @@
 		if(critter.health == critter.maxHealth)
 			to_chat(user, "<span class='notice'>[M] is at full health.</span>")
 			return
-		if(heal_brute < 1)
+		if(heal_brute < 0.1)
```
Comment (PowerfulBacon) [inline code/game/objects/items/stacks/medical.dm]: Use the define for this (minimum damage something or the other)

**Comment by ivanmixo** [comment]
> Status of this PR? Requested changes implemented?

I disputed the reviews I disagreed with and implemented the suggestions

File: code/game/objects/items/stacks/medical.dm (hunk 31 above)
Comment (ivanmixo) [reply code/game/objects/items/stacks/medical.dm]: Haven't been able to find it.

File: code/game/objects/items/stacks/medical.dm [hunk 32]
```diff
@@ -7,98 +7,90 @@
...
-					to_chat(user, "<span class='warning'>[H] isn't bleeding!</span>")
-					return
-
-
-	if(isliving(M))
-		if(!M.can_inject(user, 1))
+		if(critter.health == critter.maxHealth)
+			to_chat(user, "<span class='notice'>[M] is at full health.</span>")
 			return
+		if(heal_brute < 0.1)
```
Comment (ivanmixo) [inline code/game/objects/items/stacks/medical.dm]:
```suggestion
		if(heal_brute < DAMAGE_PRECISION)
```

File: code/game/objects/items/stacks/medical.dm [hunk 33]
```diff
@@ -7,98 +7,90 @@
...
+		to_chat(user, "<span class='warning'>[M]'s [parse_zone(user.zone_selected)] isn't hurt!")
+		return
 
-	if(iscarbon(M))
-		var/mob/living/carbon/C = M
-		affecting = C.get_bodypart(check_zone(user.zone_selected))
-		if(!affecting) //Missing limb?
-			to_chat(user, "<span class='warning'>[C] doesn't have \a [parse_zone(user.zone_selected)]!</span>")
+	if((affecting.brute_dam && !affecting.burn_dam && !heal_brute) || (affecting.burn_dam && !affecting.brute_dam && !heal_burn)) //suffer
+		to_chat(user, "<span class='warning'>This type of medicine isn't appropriate for this type of wound.")
```
Comment (itsmeow) [inline code/game/objects/items/stacks/medical.dm]: missing closing span

File: code/game/objects/items/stacks/medical.dm [hunk 34]
```diff
@@ -7,98 +7,90 @@
...
 				return
-			user.visible_message("<span class='green'>[user] applies [src] on [t_himself].</span>", "<span class='green'>You apply [src] on yourself.</span>")
+			H.suppress_bloodloss(stop_bleeding)
 
+	if(!IS_ORGANIC_LIMB(affecting))
+		to_chat(user, "<span class='warning'>Medicine won't work on a robotic limb!</span>")
+		return
+
+	if(!(affecting.brute_dam || affecting.burn_dam))
+		to_chat(user, "<span class='warning'>[M]'s [parse_zone(user.zone_selected)] isn't hurt!")
```
Comment (itsmeow) [inline code/game/objects/items/stacks/medical.dm]: missing closing span

File: code/game/objects/items/stacks/stack.dm [hunk 35]
```diff
@@ -12,21 +12,33 @@
...
 	icon = 'icons/obj/stack_objects.dmi'
 	gender = PLURAL
+	///The list recipes you can make with the stack
 	var/list/datum/stack_recipe/recipes
```
Comment (itsmeow) [inline code/game/objects/items/stacks/stack.dm:16]: these docs are fine but all of these changes are completely unrelated and don't follow good atomic PR standards. At least add to CL I guess

File: code/modules/clothing/gloves/miscellaneous.dm [hunk 36]
```diff
@@ -112,16 +112,15 @@
...
 		new	/obj/effect/temp_visual/telegloves(A.loc)
-		M.changeNext_move(CLICK_CD_MELEE)
-		if(do_after_mob(M, A, 8))
-			new /obj/effect/temp_visual/telekinesis(M.loc)
-			playsound(M, 'sound/weapons/emitter2.ogg', 25, 1, -1)
-			A.attack_hand(M)
-			return 1
-
+		user.changeNext_move(CLICK_CD_MELEE)
+		if(do_mob(user, A, 8))
```
Comment (itsmeow) [inline code/modules/clothing/gloves/miscellaneous.dm]: 0.8 SECONDS

File: code/modules/reagents/reagent_containers/patch.dm [hunk 37]
```diff
@@ -4,28 +4,28 @@
...
```
Comment (itsmeow) [inline code/modules/reagents/reagent_containers/patch.dm]: why is the super call implementing the affecting argument but this proc only has the two?

File: code/game/objects/structures/crates_lockers/closets.dm [hunk 38]
```diff
@@ -343,22 +343,21 @@
...
 	add_fingerprint(user)
-	user.visible_message("<span class='warning'>[user] [actuallyismob ? "tries to ":""]stuff [O] into [src].</span>", \
-				 	 	"<span class='warning'>You [actuallyismob ? "try to ":""]stuff [O] into [src].</span>", \
-				 	 	"<span class='italics'>You hear clanging.</span>")
+	user.visible_message("<span class='warning'>[user] [actuallyismob ? "tries to " : ""]stuff [O] into [src].</span>", \
+						"<span class='warning'>You [actuallyismob ? "try to " : ""]stuff [O] into [src].</span>", \
+						"<span class='italics'>You hear clanging.</span>")
 	if(actuallyismob)
-		if(do_after_mob(user, targets, 40))
+		if(do_mob(user, O, 40))
```
Comment (itsmeow) [inline code/game/objects/structures/crates_lockers/closets.dm]: 4 SECONDS

File: code/game/objects/items/miscellaneous.dm [hunk 39]
```diff
@@ -215,25 +215,25 @@
 	if(!proximity_flag)
 		return
 	if(isliving(target))
-		var/mob/living/M = target
-		var/kidnaptime = max(10, (M.health * (M.mob_size / 2)))
+		var/mob/living/kidnapee = target
+		var/kidnaptime = max(10, (kidnapee.health * (kidnapee.mob_size / 2)))
 		if(iscarbon(target))
 			kidnaptime += 100
```
Comment (itsmeow) [inline code/game/objects/items/miscellaneous.dm]: change these to SECONDS defines

File: code/modules/reagents/reagent_containers/patch.dm [hunk 40]
```diff
@@ -4,28 +4,28 @@
 	icon = 'icons/obj/chemical.dmi'
 	icon_state = "bandaid"
 	item_state = "bandaid"
-	possible_transfer_amounts = list()
 	volume = 40
 	apply_type = PATCH
 	apply_method = "apply"
 	self_delay = 30		// three seconds
```
Comment (itsmeow) [inline code/modules/reagents/reagent_containers/patch.dm]: 3 SECONDS

File: code/game/objects/items/stacks/medical.dm [hunk 41]
```diff
@@ -7,98 +7,90 @@
...
 	novariants = FALSE
+	///How much brute does it heal?
 	var/heal_brute = 0
+	///How much burn does it heal?
 	var/heal_burn = 0
+	///For how long does it stop bleeding?
 	var/stop_bleeding = 0
-	var/self_delay = 50
+	///How long does it take to apply on yourself?
+	var/self_delay = 20
```
Comment (itsmeow) [inline code/game/objects/items/stacks/medical.dm]: 2 SECONDS

**Review by itsmeow — CHANGES_REQUESTED** [review]

File: code/modules/clothing/gloves/miscellaneous.dm [hunk 42]
```diff
@@ -112,16 +112,15 @@
...
-	var/mob/living/M = loc
-	if(get_dist(A, M) <= 1)
-		return 0
-	if(M in viewers(range, A))
-		M.visible_message("<span_class ='danger'>[M] waves their hands at [A]</span>", "<span_class ='notice'>You begin manipulating [A].</span>")
+	var/mob/living/user = loc
+	if(get_dist(A, user) <= 1)
+		return FALSE
+	if(user in viewers(range, A))
+		user.visible_message("<span_class ='danger'>[user] waves their hands at [A]</span>", "<span_class ='notice'>You begin manipulating [A].</span>")
```
Comment (itsmeow) [inline code/modules/clothing/gloves/miscellaneous.dm]: what is this wacky `_class` stuff

Comment (ivanmixo) [reply code/modules/clothing/gloves/miscellaneous.dm]: What?

File: code/modules/reagents/reagent_containers/patch.dm (hunk 37 above)
Comment (ivanmixo) [reply code/modules/reagents/reagent_containers/patch.dm]: image

Terrible.

File: code/game/objects/items/stacks/stack.dm [hunk 43]
```diff
@@ -396,7 +405,7 @@
 		var/stackmaterial = round(input(user,"How many sheets do you wish to take out of this stack? (Maximum  [max])") as null|num)
 		max = get_amount()
 		stackmaterial = min(max, stackmaterial)
-		if(stackmaterial == null || stackmaterial <= 0 || !user.canUseTopic(src, BE_CLOSE, ismonkey(user)))
+		if(!stackmaterial || !user.canUseTopic(src, BE_CLOSE, ismonkey(user)))
```
Comment (ivanmixo) [inline code/game/objects/items/stacks/stack.dm]:
```suggestion
		if(!stackmaterial || stackmaterial < 0 || !user.canUseTopic(src, BE_CLOSE, ismonkey(user)))
```

File: code/modules/clothing/gloves/miscellaneous.dm (hunk 42 above)
Comment (itsmeow) [reply code/modules/clothing/gloves/miscellaneous.dm]: `<span_class`

Comment (ivanmixo) [reply code/modules/clothing/gloves/miscellaneous.dm]: Oh true, what the fuck?

Comment (ivanmixo) [inline code/modules/clothing/gloves/miscellaneous.dm]:
```suggestion
		user.visible_message("<span class='danger'>[user] waves their hands at [A]</span>", "<span class='notice'>You begin manipulating [A].</span>")
```

**Review by itsmeow — APPROVED** [review]

File: code/game/objects/structures/crates_lockers/closets.dm (hunk 12 above)
Comment (PowerfulBacon) [reply code/game/objects/structures/crates_lockers/closets.dm:371]: Makes merge conflicts more likely, makes reviewing harder since rather than focusing on 1 section of the code it interlinks with more random sections of the code which have no correlation and in turn causes reviews to be slower and the PRs to be way more likely to not get reviewed and just expire

Comment (ivanmixo) [reply code/game/objects/structures/crates_lockers/closets.dm:371]: Really gotta think hard about how this change affects other parts of the code, yes...

**Review by PowerfulBacon — APPROVED** [review]
